values.yaml

```diff
@@ -145,3 +151,8 @@ extraVolumes: []
 #        - key: cloudfront.pem
 #          path: cloudfront.pem
 #          mode: 511
+
+serviceMonitor:
+  enabled: false
```

Comment:
:memo: Need to document this new configuration value in the README.

Reply:
Check, also added some lines to the values.yaml